docs/pages/dev-guide/developer-guide.md

```diff
@@ -819,6 +821,8 @@ pods:
         memory: 256
 ```
 
+Currently an empty YAML dictionary is passed as the body for each network definition under `networks`, since we only support joining virtual networks by name, but in the future it will be possible to specify port mappings and other information in a network definition.
```

Comment:
I've been thinking about this, especially because there's a lot of static validation of the YAML we could do (beyond just whether all required fields are present and their types are correct) to save developers the pain of having to go through a full stub universe deployment just to find out they made a trivial error in their spec.